cluster/addons/calico-policy-controller/calico-node.yaml

```diff
@@ -0,0 +1,107 @@
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  name: calico-node
+  namespace: kube-system
+  labels:
+    kubernetes.io/cluster-service: "true"
+    k8s-app: calico-node
+spec:
+  selector:
+    matchLabels:
+      kubernetes.io/cluster-service: "true"
+      k8s-app: calico-node
+  template:
+    metadata:
+      labels:
+        kubernetes.io/cluster-service: "true"
+        k8s-app: calico-node
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+        scheduler.alpha.kubernetes.io/tolerations: |
+          [{"key": "dedicated", "value": "master", "effect": "NoSchedule" },
+           {"key":"CriticalAddonsOnly", "operator":"Exists"}]
+    spec:
+      hostNetwork: true
+      containers:
+        # Runs calico/node container on each Kubernetes node.  This
+        # container programs network policy and routes on each
+        # host.
+        - name: calico-node
+          image: calico/node:master
+          env:
+            - name: DATASTORE_TYPE
+              value: "kubernetes"
+            - name: FELIX_LOGSEVERITYSCREEN
+              value: "debug"
+            - name: CALICO_NETWORKING_BACKEND
+              value: "none"
+            - name: CALICO_IPV4POOL_CIDR
+              value: "10.244.0.0/16"
```

Comment:
This CIDR range is used by Calico to perform outgoing NAT for Pods to access the internet, etc and is required as of k8s v1.5 since the kubelet stopped doing NAT for CNI plugins.

It probably should be configurable if the cluster CIDR is configurable (they should have the same value).  I think that would mean making this file into a template.